adoc/admin-security-certificates.adoc

```diff
@@ -825,6 +825,15 @@ kubectl apply -f metrics-server-certificate.yaml
 <5> The cert-manager issuer name.
 <6> Issuer or ClusterIssuer.
 
+.Cert-Manager Known Issue
+[NOTE]
```

Comment:
Looks like user should watch out for this. Maybe better to use `WARNING` instead.

Reply:
sounds good.